docs/courses/step4-control_flow_graphs/cfg.md

```diff
@@ -0,0 +1,48 @@
+# angr courses - step 4 - control flow graphs
+
+The binary for this course can be found [here](./).
+
+##### Background: Control Flow Graphs
+The control flow graph (CFG) represents all possible paths of a program.
+Its nodes are the basic blocks and a directed edge from node _a_ to _b_ indicates that, in the binary, a jump exists from block _a_ to _b_.
+The CFG plays a central role in binary analysis, as it gives a good understanding of what a program does.
+In angr, it is used in several other analyses and is implemented twice, with the goal being on the one hand accuracy (CFGAccurate) and on the other hand speed (CFGFast).
+
```

Comment:
Reiterating my previous note, `CFGFast` vs. `CFGAccurate`, but the previous exampled used just `CFG()`